### ISSUE TITLE
Definir contenido de "Redes Sociales"

### ISSUE THREAD

**Issue by jgdev** [Somos-Trufas/sitioweb#8]

**Comment by jgdev** [comment]
@TrufaSuprema creo que para esto ¿será mejor simplemente colocarlo en el header/footer de todas las páginas?

**Comment by TrufaSuprema** [comment]
Claro, sería lo ideal. Aunque el botón de Discord sería bueno que tenga un brillo o destello encima. Igual que el de donar.
